pkg/operators/v1alpha1/clusterserviceversion_types.go

```diff
@@ -228,7 +230,7 @@ func (w *WebhookDescription) GetMutatingWebhook(namespace string, namespaceSelec
 				Name:      w.DomainName() + "-service",
 				Namespace: namespace,
 				Path:      w.WebhookPath,
-				Port:      &w.ContainerPort,
```

Comment:
Ditto

Reply:
Yes - we'll want to know when this value is actually set.

Reply:
See above.